ops-bedrock/allocate-fork-public-network.py

```diff
@@ -0,0 +1,130 @@
+import argparse
```

Comment:
Because we use the forked public network, so we can't use the genesis file for deploying our contract.

To pass this step, I overrided the balance, storage, and nonce of each contract in `alloc` on the genesis file to deploy this to the forked public network.